### PR TITLE
Add Corner sides

### PR DESCRIPTION
**Pull request by mmadjer**
Ability to apply individual corner sides to a view

### CODE REVIEW

**Review by tbaranes — APPROVED** [review]
That's a good idea! Could you also add an entry in the CHANGELOG and in the Documentation/API.md?

**Comment by mmadjer** [comment]
That should be done. Let me know if something else needs to be done.

File: IBAnimatable/CornerDesignable.swift [hunk 1]
```diff
@@ -10,12 +10,50 @@ public protocol CornerDesignable {
...
+        layer.cornerRadius = cornerRadius
+      } else {
+        layer.cornerRadius = 0.0
+
+        // if a layer mask is specified, remove it
+        layer.mask?.removeFromSuperlayer()
+
+        // configure corner for specified sides
+        let corner = CAShapeLayer()
+        corner.name = "cornerSideLayer"
```
Comment (JakeLin) [inline IBAnimatable/CornerDesignable.swift:33]: Do we need the name here?

File: IBAnimatableApp/CornerViewController.swift [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+//  IBAnimatable
+//
+//  Created by Miroslav Valkovic-Madjer on 17/11/16.
+//  Copyright © 2016 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+import IBAnimatable
+
+class CornerViewController: UIViewController {
```
Comment (JakeLin) [inline IBAnimatableApp/CornerViewController.swift:12]: It is awesome to have an example for this feature 👏

**Review by JakeLin — APPROVED** [review]
Great PR, it is awesome to update the demo app to demonstrate the new feature. Well done, 👍

**Comment by JakeLin** [comment]
@mmadjer Thank you very much for your awesome PR. Thanks for updating the CHANGELOG and API docs as well. Could you like to join us to the Github group  [IBAnimatable](https://github.com/IBAnimatable/IBAnimatable)? We can make some cool stuff together.

**Comment by mmadjer** [comment]
@JakeLin, no worries.
That sounds good. Im interested to join.

**Comment by JakeLin** [comment]
@mmadjer Great, I have sent you an invite for the joining IBAnimatable Github group. Do you use Slack? If yes, can you send me your Slack account email, then I can send you an invite to our Slack channel? You can find my email address on my profile page https://github.com/JakeLin , thanks.

**Comment by mmadjer** [comment]
Hi Jake,
my Slack account email is miro.madjer@gmail.com

**Pull request merged**